src/api/videoRoom.ts

```diff
@@ -0,0 +1,79 @@
+import Bugsnag from "@bugsnag/js";
+import firebase from "firebase/app";
+
+import { VideoChatRequest, VideoChatRequestState } from "types/VideoRoom";
+
+export const inviteToVideoChat = async (
+  hostUserId: string,
+  hostUserLocation: string,
+  invitedUserId: string
+) => {
+  const videoRoomRequest: VideoChatRequest = {
+    hostUserId: hostUserId,
+    hostUserLocation: hostUserLocation,
+    invitedUserId: invitedUserId,
+    invitedUserLocation: "",
```

Comment:
Why is this a blank string? If we don't/can't have the appropriate information here then the type should be potentially undefined, and it should be left unset here.